doc/api.md

```diff
@@ -149,6 +150,15 @@ This behavior allows that autoprovisioned parameters can freely established modi
 creation using the provisioning API. However, note that if a device (autoprovisioned or not) doesn't have these
 parameters defined at device level in database, the parameters are inherit from config group parameters.
 
+## Special measures and attributes names
+
+In case of arriving measures with name `id` or `type`, they are automatically transformed to `measure_id` and 
+`measure_type` attributes at Context Broker update. The reason behind this is to avoid to collide with the original entity id and type, as mechanism 
```

Comment:
```suggestion
`measure_type` attributes at Context Broker update. The reason behind this is to avoid to collide with the original entity ID and type, as mechanism 
```